### PR TITLE
feat: Addition of CsvSeedWriter

### PR DESCRIPTION
**Pull request by piotrmoszkowicz**
Hello experts,
As part of this PR I am willing to add `CsvSeedWriter`, which you can use to save generated seeds in `.csv` format.
This is my first contribution, so bear with me ;)

cc: @tboldagh

### CODE REVIEW

**Comment by paulgessinger** [comment]
Hey @piotrmoszkowicz!

Looks good at first glance. Would you be willing to add a python-based test for the writer?

**Comment by piotrmoszkowicz** [comment]
Hi @paulgessinger ,
Thanks for fast reply! Yes, I will definitely add the tests :)

File: Examples/Io/Csv/src/CsvSeedWriter.cpp [hunk 1]
```diff
@@ -0,0 +1,84 @@
+// This file is part of the Acts project.
+//
+// Copyright (C) 2021 CERN for the benefit of the Acts project
```
Comment (russofel) [inline Examples/Io/Csv/src/CsvSeedWriter.cpp:3]:
```suggestion
// Copyright (C) 2023 CERN for the benefit of the Acts project
```

File: Examples/Io/Csv/src/CsvSeedWriter.cpp [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+
+#include "ActsExamples/Io/Csv/CsvSeedWriter.hpp"
+
+#include "Acts/Definitions/Units.hpp"
+#include "ActsExamples/EventData/SimSeed.hpp"
+#include "ActsExamples/Utilities/Paths.hpp"
+
+#include <ios>
+#include <optional>
+#include <stdexcept>
```
Comment (niermann999) [inline Examples/Io/Csv/src/CsvSeedWriter.cpp:17]: Are all of these includes actually needed?

**Comment by andiwand** [comment]
Hi @piotrmoszkowicz ! Any updates on this one?

**Comment by paulgessinger** [comment]
Can we get this one in @piotrmoszkowicz?

**Pull request closed without merging**